### ISSUE TITLE
Type definitions for TypeScript

### ISSUE THREAD

**Issue by dartess** [Stuk/jszip-utils#34]
Hi @Stuk ! How do you feel about TypeScript? There seems to be no types for your package. Would you mind including type definitions in the npm package? I could work on it.

**Comment by Stuk** [comment]
Hi @dartess! I'm excited about TS! I would actually like to convert both the main JSZip repo and this one to use Typescript. If you'd be interested in setting up a strict TS config and doing some of the conversion I'd really appreciate that.

**Comment by dartess** [comment]
Yes, I would like to try it.  I can start next week.  Do you have any wishes?

**Comment by vtrphan** [comment]
what's the status of this? I really need TS compatibility.  Thanks

**Comment by subhash131** [comment]
hello, is there any update??
